src/api/build.gradle

```diff
@@ -27,6 +27,7 @@ dependencies {
     implementation("io.micronaut.security:micronaut-security-session")
     implementation("io.micronaut:micronaut-validation")
     implementation("io.micronaut:micronaut-session")
+    implementation("io.netty:netty-all:4.1.83.Final")
```

Comment:
See https://github.com/netty/netty/issues/12725 . This can be removed when Micronaut is upgraded to the latest Netty release.

Reply:
@dradosevic Micronaut 3.7.3 is out now with the upgrade to this  version, so perhaps you can send another PR with that upgrade and the removal of this dependency